.circleci/config.yml

```diff
@@ -4131,7 +4131,7 @@ jobs:
             git clean -fd
             # Loop to prevent failure due to maven-ant-tasks not downloading a jar..
             for x in $(seq 1 3); do
-                ${ANT_HOME}/bin/ant realclean; ${ANT_HOME}/bin/ant jar dtest-jar
+                ${ANT_HOME}/bin/ant realclean; ${ANT_HOME}/bin/ant jar dtest-jar -Dno-checkstyle=true -Drat.skip=true
```

Comment:
are these still needed if `check` is now its own target …?

Reply:
yes, because dtest jars are built for older Cassandra versions and we decided to not port this change to older branches